WooCommerce/Classes/ViewRelated/OrdersBadgeController.swift

```diff
@@ -47,6 +47,9 @@ final class OrdersBadgeController {
         case .vertical:
             xOffset = showsNinePlusText ? Constants.xOffsetForNinePlus: Constants.xOffset
             yOffset = showsNinePlusText ? Constants.yOffsetForNinePlus: Constants.yOffset
+        @unknown default:
+            xOffset = showsNinePlusText ? Constants.xOffsetForNinePlusLandscape: Constants.xOffsetLandscape
+            yOffset = showsNinePlusText ? Constants.yOffsetForNinePlusLandscape: Constants.yOffsetLandscape
```

Comment:
non-blocking question: I don't have the data offhand for phone vs. tablet and portrait vs. landscape usage, my rough guess is more phone and portrait usage. not sure if it might be worth making the default to `.vertical`? alternatively, we could create our own enum type for just `vertical` and `horizontal` cases instead of using `UIView`'s `axis` enum :)

Reply:
🤦‍♀  I meant to set it to `.vertical`, thanks for catching that! I don't know this code well enough to create a new enum, so I thought adding the smallest set of changes would be acceptable for now.